test/render/index.js

```diff
@@ -22,7 +22,7 @@ import TEST_CASES from './test-cases';
 import {WIDTH, HEIGHT} from './constants';
 import {SnapshotTestRunner} from '@deck.gl/test-utils';
 
-import './jupyter-widget';
+// import './jupyter-widget';
```

Comment:
@Pessimistress @ajduberstein 

As far as I can tell, math.gl 4.0 breaks jupyter-widget tests. The tests just freeze, I haven't figured out how to proceed with debugging

Reply:
Please add a `TODO v9` comment.

Reply:
The good news is that these tests seem to work again on 9.0, so it is presumably related to ES / CJS incompatibility causing some problems for Jupyter on 8.10.

We may need to make a decision on whether we can land this on 8.10 and temporarily break jupyter support). Since 8.10 is only dev releases and not for production this may be acceptable.

(I understand there is a new 8.0 python widgets library, maybe it addresses some of these things.)